packages/grafana-data/src/utils/logs.ts

```diff
@@ -1,6 +1,6 @@
 import { countBy, chain, escapeRegExp } from 'lodash';
```

Comment:
moved from core to grafana/data so we can use it in LogRows that is in grafana/ui